tutorials/io/data_paths.rst

```diff
@@ -125,9 +157,9 @@ Self-contained mode
 
 If you create a file called ``._sc_`` or ``_sc_`` in the same directory as the
 editor binary, Godot will enable *self-contained mode*. This mode makes Godot
-write all user data to a directory named ``editor_data/`` in the same directory
-as the editor binary. You can use it to create a portable installation of the
-editor.
+write all editor data, settings, and cache to a directory named ``editor_data/``
+in the same directory as the editor binary.
+You can use it to create a portable installation of the editor.
```

Comment:
Weird line break, should probably move a bit of the sentence to the previous line.

Reply:
> This means that future changes to the previous sentence won't necessarily cause a rewrap of the whole paragraph, because there's some whitespace available for growth.

Well, I see your point, but since we don't apply this principle consistently throughout the docs, it feels rather arbitrary. Though that could be interesting if we did. Every sentence from a new line would definitely improve readability of some diffs.

Reply:
Well I apply this principle wherever it makes sense to me. For example here:
```
- **Cache** contains data generated by the editor, or stored temporarily.
  It can safely be removed when Godot is closed.
```

"temporarily." ends at column 74, if we let ourselves go to 80 the "It" should go on the first line, but it's ugly, so I prefer to wrap earlier.

Overall strict wrapping rules are unenforceable and we have a lot of documentation which doesn't hard wrap at 80 chars, so I don't think consistency is achievable here. Everyone kind of uses what works for them as long as it's not *wildly* inconsistent.

Reply:
After further edit from bruvzg's input I had to rewrap some more so I went with this:
![image](https://user-images.githubusercontent.com/4701338/189131603-7f35dabd-9d88-4ebf-a33a-c9ac1022ee2a.png)

That could indeed be worth exploring as a recommendation for doc writers, though I'm not sure it's _always_ meaningful to wrap new sentences on the next line. Might need a recommendation that depends on the sentence length and whether it fits without wrapping after the previous sentence.

E.g. should this:
```
Hello world. This is another sentence. And now a longer one that
wraps around because it's longer.
```
be:
```
Hello world.
This is another sentence.
And now a longer one that wraps around because it's longer.
```
or
```
Hello world. This is another sentence.
And now a longer one that wraps around because it's longer.
```